packages/panzer/disc-fe/src/Panzer_EpetraVector_ReadOnly_GlobalEvaluationData.cpp

```diff
@@ -131,6 +131,18 @@ setOwnedVector(const Teuchos::RCP<const Thyra::VectorBase<double> >&
 */
 }
 
+///////////////////////////////////////////////////////////////////////////////
+//
+//  getOwnedMap()
+//
+///////////////////////////////////////////////////////////////////////////////
+Teuchos::RCP<const Epetra_Map> EpetraVector_ReadOnly_GlobalEvaluationData::
+getOwnedMap() const
+{
+  TEUCHOS_ASSERT(isInitialized_);
+  return ownedMap_;
+} // end of getOwnedMap()
+
```

Comment:
This routine was added so that we can carry the owned map around with us for when we need to convert the owned vector from a `Thyra::VectorBase<double>` to an `Epetra_Vector`.